var/spack/repos/builtin/packages/py-devito/package.py

```diff
@@ -16,51 +16,51 @@ class PyDevito(PythonPackage):
     """
 
     homepage = "https://www.devitoproject.org/"
-    pypi = "devito/devito-4.6.2.tar.gz"
+    pypi = "devito/devito-4.8.1.tar.gz"
 
-    version("4.6.2", sha256="39c2210a192ad69953b4f8d93440ffd72b07d739c4fe2290e2b182adfb7e143f")
+    version("4.8.1", sha256="56d0957a3226ed2a81c408107a614f04faa896d42c83a8b2bd1c8b1100adf51d")
 
     variant("mpi", default=False, description="Enable MPI support")
     variant("matplotlib", default=False, description="Enable matplolib support")
-    variant("ipyparallel", default=False, description="Enable ipyparallel support")
     variant("pandas", default=False, description="Enable pandas support")
```

Comment:
Would be more accurate to have a single "+optional" variant to match the pip behavior

Reply:
Good idea.